rs/execution_environment/tests/execution_test.rs

```diff
@@ -2893,3 +2893,78 @@ fn no_critical_error_on_empty_data_segment() {
         "Wasm module has invalid data segment of 0 bytes at 1.",
     );
 }
+
+#[test]
+fn failed_stable_memory_grow_cost_and_time_single_canister() {
+    const NUM_WASM_PAGES: u64 = 116 * 1024 * MIB / 65_536;
```

Comment:
Nit: Maybe you can use
`ic_replicated_state::canister_state::WASM_PAGE_SIZE_IN_BYTES` instead of `65_536`. There is as well const `GiB`. The same applies for the test below